src/main/scala/org/ergoplatform/wallet/secrets/JsonSecretStorage.scala

```diff
@@ -0,0 +1,80 @@
+package org.ergoplatform.wallet.secrets
+
+import java.io.{File, PrintWriter}
+import java.util.UUID
+
+import io.circe.generic.auto._
+import io.circe.parser._
+import io.circe.syntax._
+import org.ergoplatform.wallet.crypto
+import org.ergoplatform.wallet.settings.{EncryptionSettings, WalletSettings}
+import scorex.util.encode.Base16
+
+import scala.util.Try
+
+/**
+  * Secret storage backend. Stores encrypted seed in json file in the file system.
```

Comment:
Please specify the file format. One line == one secret? what fields does it contain?